root/artist/ArtistMerge.js

```diff
@@ -69,8 +69,23 @@ const ArtistMerge = ({$c, form, toMerge}: Props) => {
 
           <FormRowCheckbox
             field={form.field.rename}
+            help={
+              <>
+                <p>
+                  {l('You should only use the checkbox above to fix errors (e.g. typos).')}
+                </p>
+                <p>
+                  {exp.l(
+                    `If the name appears on the cover of a release, don't check the box:
+                     the artists will still be combined but the {doc_acs|artist credits}
```

Comment:
Hrm, it's not totally clear if the info after the colon is what happens if you don't check the box, or if you do. As a dev it's obvious to me from the context, but I wouldn't be so sure for a regular user trying to figure this out.

```suggestion
                     the artists will still be combined if you don’t, but the {doc_acs|artist credits}
```

Reply:
Good idea, changed.